debug: false
dist_thres: 3.6
x_max: 5.0
x_min: 1.0
y_max: 1.0
y_min: -1.0
z_max: 1.0
z_min: -1.0
